app/src/main/java/org/ieeeguc/ieeeguc/models/User.java

```diff
@@ -1,5 +1,102 @@
 package org.ieeeguc.ieeeguc.models;
 
+import org.json.JSONObject;
+
+import java.util.Date;
 public class User {
+    private  type Type;

```

Comment:
The type should start with capital however the variable name starts with a small letter